pystiche_papers/ulyanov_et_al_2016/_utils.py

```diff
@@ -62,10 +62,10 @@ def hyper_parameters(
             # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/src/descriptor_net.lua#L17
             else "bilinear",
         ),
-        batch_sampler=HyperParameters(
+        sampler=HyperParameters(
```

Comment:
Given that we don't have a `sampler` anymore, we should just remove this level to be able to do `hyper_parameters.num_samples`. The same applies to `batch_size`.